src/pagination/pagination.component.ts

```diff
@@ -231,11 +231,12 @@ export class Pagination {
 	 * @memberof Pagination
 	 */
 	get lastPage(): number {
-		return Math.ceil(this.model.totalDataLength / this.model.pageLength);
+		let last = Math.ceil(this.model.totalDataLength / this.model.pageLength);
+		return last !== 0 ? last : 1;
```

Comment:
Looks like we shouldn't return negative numbers for pages
```suggestion
		return last > 0 ? last : 1;
```